deployment/CP01CNSM_Deploy.csv

```diff
@@ -475,71 +475,71 @@ AR52,,AR61,,CP01CNSM-SBD12-04-PCO2AA000,14,1,2021-03-31T15:36:00,2021-11-03T17:0
+AR66,,,,CP01CNSM-RIC21-00-CPMENG000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,3703-00051-00001-01078,40.13325,-70.778317,,7,135,,,
+AR66,,,,CP01CNSM-RID26-00-DCLENG000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,3718-00001-00001-00068,40.13325,-70.778317,,7,135,,,
+AR66,,,,CP01CNSM-RID26-04-VELPTA000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-VELPTA-08532,40.13325,-70.778317,,7,135,,,
+AR66,,,,CP01CNSM-RID26-06-PHSEND000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-PHSEND-P0068,40.13325,-70.778317,,7,135,,,
+AR66,,,,CP01CNSM-RID26-07-NUTNRB000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-NUTNRB-01091,40.13325,-70.778317,,7,135,,,
+AR66,,,,CP01CNSM-RID26-08-SPKIRB000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-SPKIRB-00290,40.13325,-70.778317,,7,135,,,
+AR66,,,,CP01CNSM-RID27-00-DCLENG000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,3718-00001-00001-00062,40.13325,-70.778317,,7,135,,,
+AR66,,,,CP01CNSM-RID27-01-OPTAAD000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-OPTAAD-00207,40.13325,-70.778317,,7,135,,,
+AR66,,,,CP01CNSM-RID27-02-FLORTD000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-FLORTD-01214,40.13325,-70.778317,,7,135,,,
+AR66,,,,CP01CNSM-RID27-03-CTDBPC000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-CTDBPC-50109,40.13325,-70.778317,,7,135,,,
+AR66,,,,CP01CNSM-RID27-04-DOSTAD000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-DOSTAD-00388,40.13325,-70.778317,,7,135,,,
+AR66,,,,CP01CNSM-SBC11-00-CPMENG000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,3703-00051-00001-00090,40.13325,-70.778317,,0,135,CPM secondary 3703-00051-00001-00087,,
+AR66,,,,CP01CNSM-SBD11-00-DCLENG000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,3718-00001-00001-00053,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD11-01-MOPAK0000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-MOPAK0-11843,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD11-02-HYDGN0000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-HYDGN0-42330,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD11-06-METBKA000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-METLGR-00010,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD12-00-DCLENG000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,3718-00001-00001-00054,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD12-03-HYDGN0000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-HYDGN0-Z5971,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD12-04-PCO2AA000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-PCO2AA-31124,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD12-05-WAVSSA000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-WAVSSA-05781,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD12-06-METBKA000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-METLGR-00017,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD12-08-FDCHPA000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-FDCHPB-11802,40.13325,-70.778317,,-5,135,,,
```

Comment:
deployment time/location, recovery time, and deploy/water depth updates look good. configuration previously reviewed and approved.

Reply:
All updates look good. Approved.